prql-net/README.md

```diff
@@ -0,0 +1,27 @@
+# prql-net
```

Comment:
My sense is we should rename this to its full `dotnet` name. I've often seen `dotnet`, and I haven't seen `-net` used as much — is that accurate?

```suggestion
# prql-dotnet
```

Hopefully this will be a single find/replace and won't be too much work...

Reply:
I don't know, the framework is called ".NET", but using a dot isn't always possible, so sometimes "dotnet" is used, like in the `dotnet` command.

Reply:
Yes, that's why I'm suggesting we rename this everywhere to `prql-dotnet` — including the folder name. Is that OK?

Reply:
Yeah, if you prefer it that way, sure. Maybe it is more clear what it is, so it doesn't get confused with something network related.